### PR TITLE
expose intercom JS API 'showArticle' function through useIntercom

### PR DESCRIPTION
**Pull request by Jonneitapuro**
The Intercom Javascript API now supports opening an article in the Messenger widget (see: https://developers.intercom.com/installing-intercom/docs/intercom-javascript#intercomshowarticle-articleid)

This PR exposes that API through the useIntercom hook.

### CODE REVIEW

**Comment by devrnt** [comment]
Thanks, this is great!

File: playground/modules/useIntercom/useIntercom.tsx [hunk 1]
```diff
@@ -132,6 +133,10 @@ const RawUseIntercomPage = () => {
     trackEvent('invited-friend', { name: 'Russo' });
   }, [trackEvent]);
 
+  const handleShowArticle = React.useCallback(() => {
+    showArticle(123);
```
Comment (devrnt) [inline playground/modules/useIntercom/useIntercom.tsx:137]: I've created a test article in Intercom, you can replace the id with `4013997`

File: playground/modules/useIntercom/useIntercom.tsx [hunk 2]
```diff
@@ -251,6 +256,12 @@ const RawUseIntercomPage = () => {
           onClick={handleTrackEventWithMetaData}
         />
       </Item>
+      <Item>
+        <p>
+          opens an article with the given <code>article ID</code>
+        </p>
+        <Button label="Track event with metadata" onClick={handleShowArticle} />
```
Comment (devrnt) [inline playground/modules/useIntercom/useIntercom.tsx:263]: Update the button label

File: src/types.ts [hunk 3]
```diff
@@ -376,6 +377,14 @@ export type IntercomContextValues = {
    * ```
    */
   trackEvent: (event: string, metaData?: object) => void;
+  /**
+   * Opens the messenger with the specified article open
```
Comment (devrnt) [inline src/types.ts:381]:
```suggestion
   * Opens the messenger with the specified article
```

**Review by devrnt — CHANGES_REQUESTED** [review]
Looks great! Would you mind updating the `README` as well?

**Comment by Jonneitapuro** [comment]
I'll do the fixes tomorrow :)

**Comment by Jonneitapuro** [comment]
@devrnt I pushed my fixes to the requested changes. If I still missed something let me know :)

**Review by devrnt — APPROVED** [review]

**Comment by devrnt** [comment]
Thanks again, looks great!

**Comment by devrnt** [comment]
[v1.5.0](https://github.com/devrnt/react-use-intercom/releases/tag/v1.5.0)

**Pull request merged**